datashare-cli/src/test/java/org/icij/datashare/cli/CliExtensionFoo.java

```diff
@@ -23,5 +36,13 @@ public String identifier() {
     }
 
     @Override
-    public void run(Properties properties) {}
+    public void run(Properties properties) throws Exception {
+        Stream<Map.Entry<Object, Object>> props = properties.entrySet().stream();
+        if (this.fooProperties != null) {
+            props = props.filter(e -> !this.fooProperties.containsKey(e.getKey()));
+        }
+        props.forEach(
+            e -> System.out.println("extra key: " + e.getKey() + " extra value: " + e.getValue()));
```

Comment:
This is a bit unusual to me. Can you explain a little bit?

Reply:
I wanted to mock some CLI extension which needed to have some attributes injected. So I created an extension which get some properties injected by when create and outputs they diff with the properties provided at runtime on `stdout`

Reply:
Make sense, thanks!